src/ledgerwriter/src/main/java/anthos/samples/financedemo/ledgerwriter/GenerateKeyException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020, Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package anthos.samples.financedemo.ledgerwriter;
+
+public class GenerateKeyException extends RuntimeException {
+    public GenerateKeyException(String message, Exception e) {
+        super(message, e);
+    }
+}
```

Comment:
nit: do you think we should put this class in the same file as the JWTVerifierGenerator? It's a small class that's only used in one place, and the internet seems to [say it's ok](https://www.quora.com/Can-we-keep-more-than-one-class-in-a-single-java-file). But I'm not too familiar with Java best practices

Reply:
Since the exception is only being used by JWTVerifierGenerator, I think this makes sense. I added the exception as a static inner class of JWTVerifierGenerator.